core/src/main/java/com/graphhopper/storage/TurnCostExtension.java

```diff
@@ -17,6 +17,8 @@
  */
 package com.graphhopper.storage;
 
+import java.util.function.BiFunction;
```

Comment:
Note: on Android the `java.util.function` package is [available](https://developer.android.com/studio/write/java8-support.html) from API level 24 or higher.